test

```diff
@@ -0,0 +1 @@
+fruitReport.csv
```

Comment:
remove those two files, if you need test files, put them in /src/test/resouces

Reply:
not fixed
![image](https://github.com/user-attachments/assets/7aad0059-5e5d-4073-9db5-41b4b23e98a5)